lrv/database/migrations/2019_09_25_110602_add_no_comment_billing_option.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddNoCommentBillingOption extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('billingoptions', function($table) {
+			$table->integer('requires_comment')->nullable(true)->default(0);
+			$table->integer('is_external_club')->nullable(true)->default(0);
```

Comment:
I do not want to be picky but I do not think we will ever have an internal_club :)

Reply:
pick another name then

Reply:
other_club maybe